start/sw/web-app/laravel5.md

```diff
@@ -7,74 +7,76 @@ order: 3
 ## Laravel 5
 > The PHP Framework For Web Artisans
 
-Laravel is a high-level PHP Web framework 
+Laravel is a high-level PHP Web framework. We have to make sure all of these requirements are satisfied.
 
+1. PHP >= 5.5.9
+2. OpenSSL PHP Extension
+3. PDO PHP Extension
+4. Mbstring PHP Extension
+5. Tokenizer PHP Extension
 
-First we need to install PHP5+, Composer and the laravel installer.
-[Laravel](https://laravel.com/)
-[Composer](https://getcomposer.org/)
-
+#### Install Apache Web Server
+```bash
+sudo dnf install httpd
+# Setup Apache to automatically start upon system boot
+sudo systemctl enable httpd.service
+# Then to boot the server
+sudo systemctl start httpd
+# And stop with
+sudo systemctl stop httpd
+```
 
-#### Install MariaDB
+#### Install MariaDB Relational Database Server
 ```bash
-$ sudo dnf install mariadb-server
-# Then to boot the server 
-$ sudo systemctl start mariadb.service
+sudo dnf install mariadb-server
+# Setup MariaDB to automatically start upon system boot
+sudo systemctl enable mariadb
+# Then to boot the server
+sudo systemctl start mariadb
 # And stop with
-$ sudo systemctl stop mariadb.service
+sudo systemctl stop mariadb
+sudo mysql_secure_installation
 ```
-#### Install PHP
-And extras
+
+#### Install PHP and Extensions
 ```bash
-# Install php cli and needed extras
-$ sudo dnf install php-cli php-mcrypt-5.6.20 php-mysqlnd
+# Enable remi repository for PHP 7
```

Comment:
Could you please tell me why do you need to enable an external repo? Is it something optional? If it's optional, it should be marked that way.